### PR TITLE
Disable CDC cause of scylla/issues/6005

### PR DESCRIPTION
**Pull request by fruch**

### CODE REVIEW

**Comment by fruch** [comment]
@roydahan  this needs backport to branch-3.3

File: versions/3.4.0/patch [hunk 1]
```diff
@@ -34,7 +34,7 @@ index 64efcaa4..d093e2b0 100644
              else:
                  CCM_CLUSTER = CCMCluster(path, cluster_name, **CCM_KWARGS)
 -            CCM_CLUSTER.set_configuration_options({'start_native_transport': True})
-+            CCM_CLUSTER.set_configuration_options({'start_native_transport': True, 'experimental': True})
```
Comment (roydahan) [inline versions/3.4.0/patch]: I'm afarid it'll just be forgotten here..
I prefer to remove experimental completely, or have it as global parameter somehow for the entire run if it's possible.
So, we can have an entire run with experimental: True and one without it.

**Comment by fruch** [comment]
we also need to backport it to branch-4.0

**Comment by roydahan** [comment]
Backported.

**Pull request merged**